src/toast/toast.tsx

```diff
@@ -110,23 +115,45 @@ export const Toast = ({
             $fixed={fixed}
             {...otherProps}
         >
-            {renderIcon()}
-            <TextContainer>
-                {title && (
-                    <Title $type={type} weight="semibold">
-                        {title}
-                    </Title>
-                )}
-                {label && (
-                    <Description $type={type}>
-                        {!title ? (
-                            <Text.Body>{label}</Text.Body>
-                        ) : (
-                            <Text.BodySmall>{label}</Text.BodySmall>
+            <ContentWrapper>
```

Comment:
the layout can be handled like this

the overall wrapper is a flex (row) container that contains the icon + content + dismiss button

content is a flex container that contains the text content + action button. this is row on desktop/tablet and column on mobile

![Screenshot 2024-06-11 at 9 56 01 AM](https://github.com/LifeSG/react-design-system/assets/16853062/35a53c4d-e22a-436e-99f2-7025d3658f76)

Reply:
as discussed in the call, the icon needs to go with the text as it needs to be on the same line. resolving this comment